packages/remix/src/generators/error-boundary/lib/normalize-options.ts

```diff
@@ -0,0 +1,18 @@
+import {joinPathFragments, readProjectConfiguration, type Tree} from '@nx/devkit';
+import type {ErrorBoundarySchema} from "../schema";
+
+
+export function normalizeOptions(tree: Tree, schema: ErrorBoundarySchema): ErrorBoundarySchema {
+  const project = readProjectConfiguration(tree, schema.project);
+
+  const pathToRouteFile = joinPathFragments(project.root, schema.routePath);
```

Comment:
There's a utility function for resolving a route file that's more robust than this, example of use https://github.com/nrwl/nx-labs/blob/aafca4cf70d7bc5be1e6351b6e8775c169da9664/packages/remix/src/generators/meta/lib/v1.impl.ts#L9